data/coalition missions.txt

```diff
@@ -108,6 +109,7 @@ mission "Coalition: Contributor"
 		government "Coalition"
 	destination "Ring of Friendship"
 	on offer
+		log "After doing a few dozen jobs for the Coalition, they're finally wiling to let me purchase some of their technology. Doesn't sound like they'll sell me any of their weapons, though."
```

Comment:
Your not told what the benefit you are getting is until after you finish the mission so this should probably go in `on complete` as well.